src/Validator.php

```diff
@@ -43,7 +43,7 @@ public static function validateClassName(string $className, string $errorMessage
             'use', 'var', 'while', 'xor',
             'int', 'float', 'bool', 'string', 'true', 'false', 'null', 'void',
             'iterable', 'object', '__FILE__', '__LINE__', '__DIR__', '__FUNCTION__', '__CLASS__',
-            '__METHOD__', '__NAMESPACE__', '__TRAIT__',
+            '__METHOD__', '__NAMESPACE__', '__TRAIT__', 'self', 'parent',
```

Comment:
btw, all these `__*__` reserved keywords are not properly forbidden, because the check uses `in_array(strtolower($shortClassName), $reservedKeywords, true)`. We need to put them in lowercase in this array. Can you also update them ?

Reply:
I tried several constants ( like`` __*__`` ) but, in ``src/Generator.php`` a ``$className`` is replaced by :
```
$className = rtrim($fullNamespacePrefix, '\\').'\\'.Str::asClassName($name, $suffix);
```
Example :
Try ``make:entity __FILE__``, the classname: ``__FILE__`` become ``file`` and is defined in ``$reservedKeywords`` and maker-bundle not build files.
But:
Try ``make:entity __METHOD__``, the classname: ``__METHOD__`` become ``method`` and is not defined in ``$reservedKeywords`` and maker-bundle build files without fatal error, call stack and critical.
I can add ``'file', 'line', 'dir', 'method'`` but I'm not sure about the result.